test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -729,6 +729,84 @@ public void PackCommand_PackProject_SupportMultipleFrameworks()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_DoesNotIncludeContentFromDependencies_PackSuceeds()
```

Comment:
If you are trying to do the `What_UnderWhichConditions_Expectation`, I think the name would be better if it's something like: 

`PackCommand_WithDependencyThatContainsContentFiles_PackExcludesTransitiveContentFiles`

This is not perfect ofc, I don't love it, but it follows the structure. 
Right now everything about the test is covered in the `Y` of the `X_Y_Z`, X and Z are pretty much redundant.

Reply:
Changed test name to `PackProject_DependenciesWithContentFiles_NupkgExcludesContentFilesFromDependencies`